agent/custom/bbc_connection_manager.py

```
@@ -224,11 +224,10 @@ def connect_tcp(self, timeout: int = 10) -> bool:
         """建立 TCP 连接"""
         with self._state_lock:
             if self._state['connected'] and self._tcp_sock:
-                # 测试连接是否仍然有效
+                # 测试连接是否仍然有效（发送 get_status 命令）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)